v3/server/pages.py

```diff
@@ -89,11 +89,22 @@ async def basic_info():
 #    async def flash_FOO(message)
 # where FOO is one of the eight Bootstrap alert classes. See:
 #    https://getbootstrap.com/docs/5.0/components/alerts/#examples
-for _cat in ('primary', 'secondary', 'success', 'danger',
-             'warning', 'info', 'light', 'dark', ):
-    globals()[f'flash_{_cat}'] = functools.partial(quart.flash, category=_cat)
-del _cat
-
```

Comment:
This patch is subtly different because now the symbol is defined within this file. But I think this is what we want actually?